Library/Homebrew/formula.rb

```diff
@@ -730,12 +735,32 @@ def bottle_prefix
     prefix+".bottle"
   end
 
-  # The directory where the formula's installation logs will be written.
+  # The directory where the formula's installation or test logs will be written.
   # @private
-  def logs
+  def logs()
```

Comment:
Oops. Just an incomplete reversion of the prior change to `logs(log_type)`. Fixed.